css/sections/quote.postcss

```diff
@@ -65,7 +65,7 @@
 }
 
 
-blockquote { 
+:not(".socials") blockquote { 
```

Comment:
eew, not schön. What is impacted here? Instagram feed, which is not contained in an iframe?